### PR TITLE
mathmetical

### PR DESCRIPTION
**Pull request by Kashika-agrawal02**
mathmetical expression

### CODE REVIEW

**Comment by ayushshuklaintime** [comment]
When pushing a code, give a proper commit message, and add an appropriate title when raising a PR.

**Comment by ayushshuklaintime** [comment]
Remove Assignment-2 changes from this branch. Keep only Assignment-3 changes on this.

File: assignment3/mathmetical_expresiion.c [hunk 1]
```diff
@@ -0,0 +1,35 @@
+#include<stdio.h>
```
Comment (ayushshuklaintime) [inline assignment3/mathmetical_expresiion.c:1]: giving incorrect output if i give base negative

File: assignment3/mathmetical_expresiion.c [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+#include<conio.h>
+
+int main(){
+    unsigned int  B,N,M;
```
Comment (ayushshuklaintime) [inline assignment3/mathmetical_expresiion.c:5]: will unsigned int handle larger values. Check the requirement "Test your program for various edge cases (e.g., large values of N and M)."

File: assignment3/mathmetical_expresiion.c [hunk 3]
```diff
@@ -0,0 +1,35 @@
+#include<stdio.h>
+#include<conio.h>
```
Comment (ayushshuklaintime) [inline assignment3/mathmetical_expresiion.c:2]: What's the need of this header file.
